src/api.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+
+const request = axios.create({
+  baseURL: "https://nc-news-l5et.onrender.com/api",
```

Comment:
Nice to see you have created a base URL, this makes your code more readable and easy to maintain 👍